.github/workflows/run-benchmarks.yml

```diff
@@ -0,0 +1,74 @@
+name: Run benchmarks
+# ToDo: add an arbitrary label to keep this cache separate from the builds
```

Comment:
Reminder for this TODO @gpmayorga

Reply:
Push: Either remove comment or resolve. Based on the DM, there does not seem to be benchmark tagged cache solution>